projects/observability/src/pages/explorer/explorer-service.ts

```diff
@@ -1,8 +1,11 @@
 import { Injectable } from '@angular/core';
-import { forkJoinSafeEmpty, NavigationParams, NavigationParamsType } from '@hypertrace/common';
-import { Filter, FilterBuilderLookupService } from '@hypertrace/components';
+import { isEqual } from 'lodash-es';
 import { Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
+
+import { forkJoinSafeEmpty, NavigationParams, NavigationParamsType } from '@hypertrace/common';
+import { Filter, FilterBuilderLookupService } from '@hypertrace/components';
+import { SavedQuery } from '../../public-api';
```

Comment:
I think the import path should be that of the file instead of public-api. @jaywalker21 please confirm

Reply:
Good catch! The vscode auto-import added it from the public-api, but the type being imported was in the same directory. Changed ✅